superset/viz.py

```diff
@@ -2040,25 +2040,6 @@ def get_data(self, df: pd.DataFrame) -> VizData:
         return d
 
 
-class IFrameViz(BaseViz):
```

Comment:
Do we need to remove markdown components from viz.py too?

Reply:
yes i think i need to. but somehow i didn't find the right class for markup type in viz.py.... help needed!